[2023-08-15 20:27:27,658] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-08-15 20:27:27,662] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-08-15 20:27:27,663] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:27:27,663] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-15 20:27:27,663] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:27:27,690] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta_semana> on 2023-01-02 00:00:00+00:00
[2023-08-15 20:27:27,693] {standard_task_runner.py:52} INFO - Started process 50655 to run task
[2023-08-15 20:27:27,697] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta_semana', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/dados_climatics.py', '--cfg-path', '/tmp/tmpxbcecnc4', '--error-file', '/tmp/tmpjcp0v1ue']
[2023-08-15 20:27:27,698] {standard_task_runner.py:80} INFO - Job 133: Subtask cria_pasta_semana
[2023-08-15 20:27:27,745] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-02T00:00:00+00:00 [running]> on host ubuntu.internal.cloudapp.net
[2023-08-15 20:27:27,810] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta_semana
AIRFLOW_CTX_EXECUTION_DATE=2023-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-02T00:00:00+00:00
[2023-08-15 20:27:27,811] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-15 20:27:27,811] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/guilherme/airflow/semana=2023-01/2023-01-09"']
[2023-08-15 20:27:27,818] {subprocess.py:85} INFO - Output:
[2023-08-15 20:27:27,820] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-15 20:27:27,856] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta_semana, execution_date=20230102T000000, start_date=20230815T202727, end_date=20230815T202727
[2023-08-15 20:27:27,909] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-15 20:27:27,927] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-08-15 20:39:10,056] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-08-15 20:39:10,064] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-08-15 20:39:10,064] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:39:10,064] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-08-15 20:39:10,064] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-15 20:39:10,100] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta_semana> on 2023-01-02 00:00:00+00:00
[2023-08-15 20:39:10,104] {standard_task_runner.py:52} INFO - Started process 55612 to run task
[2023-08-15 20:39:10,110] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'cria_pasta_semana', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/dados_climatics.py', '--cfg-path', '/tmp/tmplgpkt5ih', '--error-file', '/tmp/tmp8ziavzb0']
[2023-08-15 20:39:10,110] {standard_task_runner.py:80} INFO - Job 234: Subtask cria_pasta_semana
[2023-08-15 20:39:10,169] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.cria_pasta_semana scheduled__2023-01-02T00:00:00+00:00 [running]> on host ubuntu.internal.cloudapp.net
[2023-08-15 20:39:10,281] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=cria_pasta_semana
AIRFLOW_CTX_EXECUTION_DATE=2023-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-02T00:00:00+00:00
[2023-08-15 20:39:10,282] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-08-15 20:39:10,282] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/guilherme/airflow/dados/semana=2023-01/2023-01-09"']
[2023-08-15 20:39:10,290] {subprocess.py:85} INFO - Output:
[2023-08-15 20:39:10,293] {subprocess.py:96} INFO - Command exited with return code 0
[2023-08-15 20:39:10,326] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=cria_pasta_semana, execution_date=20230102T000000, start_date=20230815T203910, end_date=20230815T203910
[2023-08-15 20:39:10,362] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-15 20:39:10,380] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
